docs/CONFIG.md

```diff
@@ -217,7 +217,7 @@ make config.img; make installer.raw
 
 ### Creating USB sticks
 
-The [scripts/mkusb.sh](../scripts/mkusb.sh) can run on Linux to create a USB stick.
+The [tools/makeusbconf.sh](../tools/makeusbconf.sh) can run on any system that supports Docker to create a USB stick.
```

Comment:
Technically it now only creates an image file and not a USB stick. I think it makes sense to provide some examples of how to put this on a USB stick for different OSs.

Reply:
> Renaming a file in the same commit as doing some significant changes makes it very painful to review. I can already see that you lost the copyright and license statements and I assume that was not an intentional change.

I am actually not quite sure why git thinks it is a rename -- I removed the old script and created a new one pretty much from scratch.

Good catch on copyright and license statements -- we need to automated scanning for those in yetus, btw.

Reply:
> Technically it now only creates an image file and not a USB stick.

Actually, it can create it just fine -- you simply need to pass /dev/XXX as an argument. Now, it will only work reliably on Linux, but given that the original script only worked Linux to begin with it doesn't seem like too big of a regression.

That said, @nordmark -- I was going to ask you to actually test it on your Linux box. Can you do that?